docs/topics/migrations.txt

```diff
@@ -276,6 +276,14 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
+
+.. note::
+
+    You may add an ``initial = True`` class property to the ``Migration`` class
+    to have Django apply the "initial migration" logic when the
```

Comment:
I think it gets hard to understand if it's kept as one sentence, might be better to have something like:

Django will fake the migration if these conditions are met: --fake-migrations arg is used, `initial = True` class property is added to migration class, and either of the two conditions are met: 1. migration creates a new table that already exists 2. migration adds new fields that already exist.

It may be more readable as a bullet list, especially because of the nested 'OR' sublist.

The tradeoff I see here is that this is a corner case that majority of readers don't need so does it need to take up so much real estate on this page?